example/sync/main.go

```diff
@@ -57,9 +57,12 @@ func run() (err error) {
 		}
 		switch choice {
 		case 1:
-			if err := connector.Sync(); err != nil {
+			replicated, err := connector.Sync()
+			if err != nil {
 				return err
 			}
+
+			fmt.Println("%d frames synced", replicated.frames_synced)
```

Comment:
I get this error when I try to build

```
# github.com/tursodatabase/go-libsql/example/sync
sync/main.go:65:47: replicated.frames_synced undefined (type libsql.replicated has no field or method frames_synced)
```

Reply:
Because frames_synced is private. You need to make the names of those fields start with a capital letters to be public. You probably want to make the name of the struct public too by starting its name with a capital letter too.